native/cocos/bindings/manual/jsb_websocket_server.cpp

```diff
@@ -147,31 +144,31 @@ static bool WebSocketServer_listen(se::State &s) {
         }
     }
 
-    WebSocketServer::listenAsync(*cobj, arg_port, arg_host, arg_callback);
+    cc::network::WebSocketServer::listenAsync(*cobj, argPort, argHost, argCallback);
```

Comment:
Need to root the server js object when successfully listen a port and unroot on close

Reply:
already root/unroot in constructor and onconnection